yaml.go

```diff
@@ -283,3 +283,34 @@ func RegisterCustomUnmarshaler[T any](unmarshaler func(*T, []byte) error) {
 		return unmarshaler(v.(*T), b)
 	}
 }
+
+// RawMessage is a raw encoded YAML value. It implements [Marshaler] and
+// [Unmarshaler] and can be used to delay YAML decoding or precompute a YAML
```

Comment:
Unless I'm wrong, these are the interfaces you should mention

```suggestion
// RawMessage is a raw encoded YAML value. It implements [BytesMarshaler] and
// [BytesUnmarshaler] and can be used to delay YAML decoding or precompute a YAML
```

Also, you could complete the documentation with something like this

```suggestion
// RawMessage is a raw encoded YAML value. It implements [BytesMarshaler] and
// [BytesUnmarshaler] and can be used to delay YAML decoding or precompute a YAML
// It also implements [json.Marshaler] and [json.Unmarshaler]
```

Feel free to adapt

Reply:
You are correct, whoops.